### PR TITLE
Rework how the toolchains are explained

### PR DESCRIPTION
**Pull request by athei**
The changes I promised [here](https://github.com/substrate-developer-hub/substrate-node-template/pull/99#issuecomment-714308392).

It sure needs some editing. Not a native speaker.

@danforbes @wheresaddie

### CODE REVIEW

File: docs/knowledgebase/getting-started/index.md [hunk 1]
```diff
@@ -80,61 +81,75 @@ rustup default stable
...
-_itself_) should use a specific Rust nightly version that is known to be compatible with the version
-of Substrate they are using. Use Rustup to install the correct nightly:
+Because the nightly toolchain is a moving target and receives daily changes the chance
+that some of them break the substrate build from time to time is non negligible.
+
+Therefore it is advised to use a fixed nightly version rather than the latest one to
+build the runtime. You can install a specific version using this command:
 
 ```bash
 rustup install nightly-<yyyy-MM-dd>
```
Comment (thiolliere) [inline docs/knowledgebase/getting-started/index.md:95]: shouldn't we say here to install the target as well:
```
rustup target add wasm32-unknown-unknown --toolchain nightly-<yyyy-MM-dd>
```

**Review by wheresaddie — CHANGES_REQUESTED** [review]
updated grammar and context terminology

File: docs/knowledgebase/getting-started/index.md [hunk 2]
```diff
@@ -3,24 +3,19 @@ title: Installation
...
 ### Unix-Based Operating Systems
 
-Substrate development is easiest on Unix-based operating systems, like macOS or Linux. The examples
+Substrate development is optimized for Unix-based operating systems like MacOS or Linux. The examples
 in the Substrate [Tutorials](../../../../tutorials) and [Recipes](https://substrate.dev/recipes/)
-use Unix-style terminals to demonstrate how to interact with Substrate from the command line.
+use the terminal to demonstrate how to interact with Substrate from the command line.
 
-#### macOS
+#### MacOS
```
Comment (athei) [inline docs/knowledgebase/getting-started/index.md:18]: macOS with a small **m** is actually correct. See: https://en.wikipedia.org/wiki/MacOS#macOS

File: docs/knowledgebase/getting-started/index.md [hunk 3]
```diff
@@ -3,24 +3,19 @@ title: Installation
...
-toolchain.
+As Substrate is built with [the Rust programming language](https://www.rust-lang.org/), the first
+thing you will need to do is prepare the development enviroment, these steps will vary based
+on the computer's operating system. You can utilize helpful utilities from the Rust toolchain
+to configure the Rust development environment. Note that these steps will be the same for all 
+Unix-based operating systems discussed on this page.
 
 ### Unix-Based Operating Systems
 
-Substrate development is easiest on Unix-based operating systems, like macOS or Linux. The examples
```
Comment (athei) [inline docs/knowledgebase/getting-started/index.md:19]: macOS with a small m is actually correct. See: https://en.wikipedia.org/wiki/MacOS#macOS

File: docs/knowledgebase/getting-started/index.md (hunk 2 above)
Comment (wheresaddie) [reply docs/knowledgebase/getting-started/index.md:18]: thanks, please revert

File: docs/knowledgebase/getting-started/index.md (hunk 3 above)
Comment (wheresaddie) [reply docs/knowledgebase/getting-started/index.md:19]: please revert and then push PR thanks

**Comment by athei** [comment]
You requested changes and made that changes yourself?

**Review by wheresaddie — APPROVED** [review]

**Comment by danforbes** [comment]
Many of these changes are being reverted in #772 - feel free to comment on that PR if you feel strongly that things need to change. My apologies for missing this PR.

**Pull request merged**